### DISCUSSION TITLE
how to account for the dc and ac ohmic loss when using pvsystem.Array and pvsystem.PVSystem ?

### DISCUSSION THREAD

**Discussion by saikatghosh90** [pvlib/pvlib-python#1845] [Q&A]
array_dcoutput = pvsystem.Array(mount=sat_mount,modules_per_string=module_series, albedo= albedo, module_type= module, strings=total_strings,
                       module_parameters=cec_module,
                       temperature_model_parameters=temp_model_parameters)

how to use - pvlib.pvsystem.dc_ohms_from_percent ?

**Comment by cwhanse** [comment]
You would use both `pvlib.pvsystem.dc_ohms_from_percent` (first) to calculate the resistance, then `pvlib.pvsystem.dc_ohmic_los`s to calculate the power loss (in W). `dc_ohms_from_percent` requires an input **dc_ohmic_percent**, and returns the resistance needed for input to `dc_ohmic_loss`. We haven't set a default for **dc_ohmic_percent** because this is relatively immature and not well documented as a model. The PVsyst default value is 1.5%.
